homeassistant/components/nws/translations/en.json

```diff
@@ -15,9 +15,9 @@
                     "longitude": "Longitude",
                     "station": "METAR station code"
                 },
-                "description": "If a METAR station code is not specified, the latitude and longitude will be used to find the closest station.",
+                "description": "If a METAR station code is not specified, the latitude and longitude will be used to find the closest station. For now, an API Key can be anything. It is recommended to use a valid email address.",
```

Comment:
My impression is the pattern is to update ../strings.json, then the stuff in translations/ is created from that.  https://developers.home-assistant.io/docs/internationalization/core/#introducing-new-strings

Can you update that instead or do you have a specific reason to update just the english translation only?